python/paddle/optimizer/rmsprop.py

```diff
@@ -160,11 +183,20 @@ def __init__(self,
         self._epsilon = epsilon
         self._momentum = momentum
         self._centered = centered
+        self.default_dict = {
+            'rho': rho,
+            'epsilon': epsilon,
+            'momentum': momentum,
+            'centered': centered,
+        }
```

Comment:
the default dict will override those in Base class?

Reply:
sure, the default dict contains the extra argument